lib/src/feedback_bottom_sheet.dart

```diff
@@ -8,26 +10,67 @@ class FeedbackBottomSheet extends StatelessWidget {
     Key? key,
     required this.feedbackBuilder,
     required this.onSubmit,
+    required this.collapsedHeight,
   }) : super(key: key);
 
   final FeedbackBuilder feedbackBuilder;
   final OnSubmit onSubmit;
+  final double collapsedHeight;
 
   @override
   Widget build(BuildContext context) {
-    // We need to supply a navigator so that the contents of the bottom sheet
-    // have access to an overlay (overlays are used by many material widgets
-    // such as `TextField` and `DropDownButton`.
-    // Typically, the navigator would be provided by a `MaterialApp`, but
-    // `BetterFeedback` is used above `MaterialApp` in the widget tree so that
-    // the nested navigation in navigate mode works properly.
-    return Navigator(
-      onGenerateRoute: (_) => MaterialPageRoute<void>(
-        builder: (context) => Material(
-          color: FeedbackTheme.of(context).feedbackSheetColor,
-          child: feedbackBuilder(context, onSubmit),
-        ),
+    return SafeArea(
+      child: DraggableScrollableSheet(
+        minChildSize: collapsedHeight / MediaQuery.of(context).size.height,
+        initialChildSize: collapsedHeight / MediaQuery.of(context).size.height,
+        builder: (context, scrollController) {
+          // We need to supply a navigator so that the contents of the bottom
+          // sheet have access to an overlay (overlays are used by many material
+          // widgets such as `TextField` and `DropDownButton`.
+          // Typically, the navigator would be provided by a `MaterialApp`, but
+          // `BetterFeedback` is used above `MaterialApp` in the widget tree so
+          // that the nested navigation in navigate mode works properly.
+          return ClipRRect(
+            borderRadius: const BorderRadius.all(
+              Radius.circular(
+                20,
+              ),
+            ),
+            child: Material(
+              color: FeedbackTheme.of(context).feedbackSheetColor,
+              child: Column(
+                children: [
+                  SingleChildScrollView(
+                    controller: scrollController,
+                    child: Center(
+                      child: Container(
```

Comment:
Is this a drag handle? I feel like this could be private class with a name. It makes it more readable and easier to understand.

Reply:
Done!